examples/README.md

```
@@ -0,0 +1,21 @@
+# Examples
+
+The example program `main` uses ystdlib's 4 different library targets.
+
+## Building
+
+To build the examples, run one of the following task commands
+```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)